config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -1342,6 +1342,21 @@ public HttpSecurity authorizeHttpRequests(
 		return HttpSecurity.this;
 	}
 
+	/**
+	 * get authorizeHttpRequests with default values
```

Comment:
Will you please use the Javadoc method description from `authorizeHttpRequests(Customizer)`?